src/database/database.ts

```diff
@@ -17,9 +18,9 @@ export const insertVtBooking = async function (): Promise<string[]> {
   vehicleBooking.FK_VEH_SYST_NO = 1234567;
   vehicleBooking.COUNTED_AXLES = 2;
 
-  const insertResult: string[] = await dbConnection()
-    .insert([vehicleBooking], ['VEHICLE_BOOKING_NO'])
-    .into('VEHICLE_BOOKING');
+  const connection: Knex<any, unknown[]> = await dbConnection();
```

Comment:
Can we maybe look at adding an any warning suppression here to hide it from future PRs as I think it comes back everytime?

Reply:
Probably one to look into, but I believe first type of the generic is meant to be the type of your data in your SQL statement, but in this case we're just creating a connection, so I don't believe we can really cast it to a type other than `any` or `unknown`.

It's only when we're using the insert statement can we really cast that type. Might be better to cast it to `unknown` for the time being